src/ng-currency.directive.js

```diff
@@ -14,6 +14,9 @@ export default function ngCurrency($filter, $locale, $timeout) {
       let initialized, hardCap, min, max, currencySymbol, ngRequired;
       let active = true;
       let fraction = 2;
+      const currencyPatterns = $locale.NUMBER_FORMATS.PATTERNS[1];
+      currencyPatterns.negPre = '(\u00a4';
+      currencyPatterns.negSuf = ')';
```

Comment:
@aguirrel 
angular/angular.js#12870
Angular 1.4.x+ changed how the currency filter works. 
Rather then negative values becoming `($1.00)` they become `-$1.00`.

Personally I feel like we should follow Angulars lead on this and use `-$1.00`.

Users can always modify the locale pattern to use parenthesis, 
however if we modify it for them then they don't get a choice.